src/contour/TerminalWindow.h

```diff
@@ -25,7 +25,7 @@
 #include <QtCore/QPoint>
 #include <QtCore/QTimer>
 #include <QtGui/QOpenGLExtraFunctions>
-#include <QtGui/QOpenGLWindow>
+#include <QOpenGLWindow>
```

Comment:
Don't they namespace per directory anymore? If the rest is still working via subdir, maybe we can prefix the include here by qt module, too?

Reply:
> Don't they namespace per directory anymore?

Yes.
`QOpenGLShader`, `QOpenGLShaderProgram` and `QOpenGLWindow` now in `/usr/include/qt6/QtOpenGL`.

Reply:
Interestingly, prefixes are rarely used in `/usr/share/doc/qt6/examples`. :)

Reply:
I do prefer them though in order to know what modules they are coming from. Qt examples don't use them most likely to express simplicity, whereas I prefer descriptability.